framework/source/class/qx/test/lang/String.js

```diff
@@ -213,6 +213,37 @@ qx.Class.define("qx.test.lang.String",
       this.assertEquals('"abc \\"defg\\" hij"', qx.lang.String.quote('abc "defg" hij'));
       this.assertEquals('"abc \\\\defg\\\\ hij"', qx.lang.String.quote('abc \\defg\\ hij'));
       this.assertEquals('"abc \\"defg\\\\ hij"', qx.lang.String.quote('abc "defg\\ hij'));
+    },
+
+    testTrim : function()
+    {
+      var str = "     foo bar     ";
+
+      this.assertIdentical(qx.lang.String.trimLeft(str), "foo bar     ");
+      this.assertIdentical(qx.lang.String.trimRight(str), "     foo bar");
+    },
+
+    testStripScripts : function()
+    {
+      var str = "This is a <script>foobar</script>test";
+
+      this.assertIdentical(qx.lang.String.stripScripts(str), "This is a test");
+
+      var nativeGlobalEval = qx.lang.Function.globalEval;
+      var called = false;
+      
+      // Patch to check if the function has been called by the stripScripts function
+      qx.lang.Function.globalEval = function(str) {
+          nativeGlobalEval(str);
+          called = true;
+      };
+
```

Comment:
You can use the build-in spy functionality to cover this use case, like:

`var spy = this.spy(qx.lang.Function, "globalEval');`

and check if it has been called with:

`this.assertCalledOnce(spy);`

and restore everything at the end:

`spy.restore()`

You have to include the qx.dev.unit.MMock Mixin to have the spy + assertion availabe in your test case.